src/lib/components/metrics-definition/MetricsDefinitionExploreMetricsButton.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import Tooltip from "$lib/components/tooltip/Tooltip.svelte";
+  import TooltipContent from "$lib/components/tooltip/TooltipContent.svelte";
+  import { getMetricsDefReadableById } from "$lib/redux-store/metrics-definition/metrics-definition-readables";
+  import { getMeasuresByMetricsId } from "$lib/redux-store/measure-definition/measure-definition-readables";
+  import { getDimensionsByMetricsId } from "$lib/redux-store/dimension-definition/dimension-definition-readables";
+  import { dataModelerService } from "$lib/application-state-stores/application-store";
+  import { EntityType } from "$common/data-modeler-state-service/entity-state-service/EntityStateService";
+  import ExploreIcon from "$lib/components/icons/Explore.svelte";
+
+  $: selectedMetricsDef = getMetricsDefReadableById(metricsDefId);
+  $: measures = getMeasuresByMetricsId(metricsDefId);
+  $: dimensions = getDimensionsByMetricsId(metricsDefId);
+
+  export let metricsDefId: string;
+
+  let tooltipText = "";
+  let buttonDisabled = true;
+  let buttonStatus = "OK";
+  $: if (
+    $selectedMetricsDef?.sourceModelId === undefined ||
+    $selectedMetricsDef?.timeDimension === undefined
+  ) {
+    tooltipText = "";
+    buttonDisabled = true;
+    buttonStatus = "MISSING_MODEL_OR_TIMESTAMP";
+  } else if ($measures.length === 0 || $dimensions.length === 0) {
+    tooltipText = "";
+    buttonDisabled = true;
+    buttonStatus = "MISSING_MEASURES_OR_DIMENSIONS";
+  } else {
+    tooltipText = undefined;
+    buttonDisabled = false;
+  }
+</script>
+
+<Tooltip location="right" alignment="middle" distance={5}>
```

Comment:
Ah, on my screen the tooltip is on the left, which is annoying since the "quick start" button's tooltip is on the right. Must be some logic in the `Tooltip` component that's causing it to flip.

Reply:
Well, it's just happening on a certain small-sized screen. So it's expected behavior.